includes/custom-metaboxes/cmb2-blog-fields.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+$blog_quote = new_cmb2_box([
+  'id' => 'blog_quote',
+  'title' => __('Blog Quote', 'imi'),
+  'object_types' => ['blog'],
+  'context' => 'normal',
+  'priority' => 'high',
+  'show_names' => true
+]);
+
+$blog_quote->add_field([
+  'desc' => __('Enter text to be displayed as quote'),
+  'id' => $prefix . 'blog_quote_text',
+  'type' => 'text'
+]);
+
+$blog_quote_author = new_cmb2_box([
```

Comment:
Just so you know you don't need to set up a new_cmb2_box for each field... so you could delete lines 18 - 25  and just add another field to the blog quote box you already set up. So what is currently line 27 would just become `$blog_quote->add_field([` . You can add as many fields as you want to a cmb2 box. Each box will be like its own section when you're viewing the wordpress backend.